pyproject.toml

```
@@ -43,10 +44,11 @@ testpaths = "cf_units"
 
 [tool.setuptools_scm]
 write_to = "cf_units/_version.py"
+local_scheme = "dirty-tag"
 
 [tool.black]
 line-length = 79
-target-version = ['py37', 'py38', 'py39']
+target-version = ["py38", "py39"]
```

```suggestion
target-version = ["py38", "py39", "py310"]
```

Unless there's a reason this was omitted here?

---

As yet, it's not supported and including it causes an issue

---

In that case it's rather odd that we're testing against Python `v3.10` - easy to misunderstand. Please could we either:

* Remove other references to `v3.10`?
  OR
* Stick an explanatory note here?

---

@trexfeathers You've got to love open-source development... `py310` is available now :rocket: